src/controllers/playback/queue/index.js

```diff
@@ -1,19 +1,49 @@
 import RemoteControl from '../../../components/remotecontrol/remotecontrol';
+import { playbackManager } from '../../../components/playback/playbackmanager';
 import libraryMenu from '../../../scripts/libraryMenu';
 import '../../../elements/emby-button/emby-button';
 
 export default function (view) {
     const remoteControl = new RemoteControl();
     remoteControl.init(view, view.querySelector('.remoteControlContent'));
+
+    let currentPlayer;
+
+    function onKeyDown(e) {
+        if (e.keyCode === 32 && e.target.tagName !== 'BUTTON') {
+            playbackManager.playPause(currentPlayer);
+            e.preventDefault();
+            e.stopPropagation();
+        }
+    }
+
+    function releaseCurrentPlayer() {
+        const player = currentPlayer;
+        if (player) currentPlayer = null;
+    }
+
+    function bindToPlayer(player) {
+        if (player !== currentPlayer) {
+            releaseCurrentPlayer();
+            currentPlayer = player;
+            if (!player) return;
```

Comment:
This seems unnecessary?

Reply:
Yeah it can be removed, I copied a lot of this and it must have slipped through.